receiver/hostmetricsreceiver/config.go

```diff
@@ -25,6 +26,13 @@ type Config struct {
 	Scrapers                       map[string]internal.Config `mapstructure:"-"`
 	// RootPath is the host's root directory (linux only).
 	RootPath string `mapstructure:"root_path"`
+
+	// Collection interval for metadata.
+	// Metadata of the particular entity in the cluster is collected when the entity changes.
+	// In addition metadata of all entities is collected periodically even if no changes happen.
+	// Setting the duration to 0 will disable periodic collection (however will not impact
+	// metadata collection on changes).
+	MetadataCollectionInterval time.Duration `mapstructure:"metadata_collection_interval"`
```

Comment:
Is there a default value for `MetadataCollectionInterval` or you have to specify it explicitly in the config to enable periodic collection? I think I would prefer to default to periodic collection with some reasonably large interval (e.g. 5 min).